### PR TITLE
fix: update hardhat dependencies and fix test-suite error codes

### PR DESCRIPTION
**Pull request by kartojal**
### Changes
- Update NPM dependencies related with hardhat
- Delete unused NPM dependencies
- Fix fake positives at revert messages

### Context
After performing maintenance of NPM dependencies and upgrading from waffle and old chai matchers to `hardhat-toolbox` tooling suite, noticed at the test suite that some matchers where not failing correctly, resulting in fake positives.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,2 +1 @@
-@aave:registry=https://npm.pkg.github.com
 engine-strict=true
```
Comment (miguelmtzinf) [inline .npmrc:1]: Should we add `save-exact=true`?

File: package.json [hunk 2]
```diff
@@ -86,8 +76,6 @@
   ],
   "license": "BUSL-1.1",
   "dependencies": {
-    "@nomiclabs/hardhat-etherscan": "^2.1.7",
-    "axios-curlirize": "^1.3.7",
     "tmp-promise": "^3.0.2"
```
Comment (miguelmtzinf) [inline package.json:79]: let's get rid of this dependency too

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aave/core-v3",
-  "version": "1.16.2-beta.1",
+  "version": ""1.16.2-beta.1",
```
Comment (miguelmtzinf) [inline package.json:3]: seems wrong

File: package.json (hunk 2 above)
Comment (kartojal) [reply package.json:79]: yes

**Review by miguelmtzinf — APPROVED** [review]

**Pull request merged**